### PR TITLE
[WebKit] Update to iOS 10.3 beta 1

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/wkwebkit.cs [hunk 1]
```diff
@@ -176,6 +176,10 @@ interface WKPreferences : NSCoding {
 
 		[Export ("plugInsEnabled")]
 		bool PlugInsEnabled { get; set; }
+
+		[Mac (10,12,3, onlyOn64 : true)]
```
Comment (VincentDondain) [inline src/wkwebkit.cs]: Likely 10.12.4 instead of 10.12.3

Comment (spouliot) [inline src/wkwebkit.cs]: if headers really say 10.2.3 then let's try it and confirm it works (and put a code comment if not). This is a kind of error that's too common in the headers :)

**Comment by dalexsoto** [comment]
Yep unavailable on macOS 10,12,3 so changed it to 10,12,4

**Comment by dalexsoto** [comment]
monotouch-test - https://bugzilla.xamarin.com/show_bug.cgi?id=52065

**Pull request merged**